test/langtools/jdk/javadoc/doclet/testPreview/TestPreview.java

```diff
@@ -121,6 +138,24 @@ public void testPreviewAPIJavadoc() {
                     <div class="block">Returns the value of the <code>i</code> record component.</div>
                     </div>
                     """);
+
+        // 8325325: Breadcrumb navigation links should not contain PREVIEW link
```

Comment:
The code is OK, but sometimes, when output should not contain something, it may be useful to have a broadly applicable "negative" call for `checkOutput`, as in `checkOutput(file, false, ...)`.   But agreed such checks can be fragile, and it is also good to have positive checks about what the output should be.